deltacat/aws/redshift/model/manifest.py

```diff
@@ -206,6 +234,8 @@ def of(
         mandatory: bool = True,
         uri: Optional[str] = None,
         uuid: Optional[str] = None,
+        content_file_category: Optional[ContentFileCategory] = ContentFileCategory.DATA,
```

Comment:
Manifest also needs to have type as spec shouldn't allow mixing DATA and POSITION_DELETE entries into a single manifest? Otherwise, it would lead a very different compactor algo.